README.md

```diff
@@ -1,15 +1,15 @@
-WORK IN PROGRESS
-================
+# Cloud Run Events
 
-Ref: https://cloud.google.com/run/docs/tutorials/pubsub
+Cloud Run Events builds on Kubernetes to enable easy consumption of events from
+GCP services. It can be useful independently, or mixed with
```

Comment:
I think the idea is to use Google Cloud instead of GCP... all over the place... 
See here: https://github.com/knative/eventing/issues/1214#issuecomment-492816301

Reply:
Done.